kamailio/local.cfg

```diff
@@ -11,6 +11,7 @@ debug = L_INFO
 #!trydef NAT-TRAVERSAL-ROLE
 #!trydef REGISTRAR-ROLE
 #!trydef PRESENCE-ROLE
+#!trydef MESSAGE-ROLE
```

Comment:
New features should be disabled so after updates system admins have to opt in, otherwise they get surprised by new behavior.